pkgs/tools/networking/mitmproxy/default.nix

```diff
@@ -21,6 +21,8 @@ python3Packages.buildPythonPackage rec {
     })
   ];
 
+  postPatch = "sed -i 's|<4.5|<4.6|' setup.py";
```

Comment:
Can you also include the package name here? Otherwise it is not clear, which version constraint you fix here.
A patch file would even better.

Reply:
I added a patch.